SpecRunner.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Jasmine Spec Runner v2.3.0</title>
+
+    <link rel="shortcut icon" type="image/png" href="lib/jasmine-2.3.0/jasmine_favicon.png">
+    <link rel="stylesheet" href="lib/jasmine-custom.css">
+
+    <script src="lib/jasmine-2.3.0/jasmine.js"></script>
+    <script src="lib/jasmine-2.3.0/jasmine-html.js"></script>
+    <script src="lib/jasmine-2.3.0/boot.js"></script>
+
+    <!-- include source files here... -->
+    <script src="src/Frame.js"></script>
+    <script src="src/Scorecard.js"></script>
+
+    <!-- include spec files here... -->
+    <script src="spec/BowlingSpec.js"></script>
```

Comment:
Rename this to `FrameSpec.js` - it's kinda a unit test anyway (because you haven't done any 'feature testing' through an interface)